README.md

```diff
@@ -1,2 +1,158 @@
 # orchestrator-helm-chart
-Helm chart to deploy the Orchestrator solution suite, including Janus IDP backstage, SonataFlow Operator, OpenShift Serverless Operator,  Knative Eventing, Knative Serving, Data Index and Job Service.
+Helm chart to deploy the Orchestrator solution suite on OpenShift, including Janus IDP backstage, SonataFlow Operator, OpenShift Serverless Operator, Knative Eventing, Knative Serving, Data Index and Job Service.
+
+This chart will deploy the following on the target OpenShift cluster:
+  - Janus IDP backstage
+  - SonataFlow Operator
+  - OpenShift Serverless Operator
+  - Knative Eventing
+  - Knative Serving
+  - Data Index Service
+  - Job Service
+  - PostgreSQL Databases 
+  - Sample workflow (event-timeout)
+
+## Installation
+
+### From source
+```console
+$ git clone git@github.com:parodos-dev/orchestrator-helm-chart.git
+$ cd orchestrator-helm-chart/charts
+$ helm dep update orchestrator
+$ helm install orchestrator orchestrator
+NAME: orchestrator
+LAST DEPLOYED: Thu Nov  2 09:02:29 2023
+NAMESPACE: jianrong
+STATUS: deployed
+REVISION: 1
+$ oc wait --for=condition=Available deploy sonataflow-operator-controller-manager -n openshift-operators
```

Comment:
FYI I had an error for a few minutes before I could have this command running:
```
$ oc wait --for=condition=Available deploy sonataflow-operator-controller-manager -n openshift-operators
Error from server (NotFound): deployments.apps "sonataflow-operator-controller-manager" not found
```

Reply:
I had this only once, I do not have it anymore

Reply:
happens to me as well. It happens if you try the command before the operator starts its installation.
perhaps there is a wait to wait on other resources first?

Reply:
I updated the instructions. The rollout process can take 10-20 min.